app/views/layouts/users/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>List of all the users</h1>
```

Comment:
- [ ] I figured that your users folder is inside the layouts, kindly note that the layout folder is a helper folder, and it users folder has to be outside it just as your posts folder is outside so that they can have access to the controller.